client/asset/btc/btc.go

```diff
@@ -861,7 +861,7 @@ var _ asset.LogFiler = (*ExchangeWalletSPV)(nil)
 var _ asset.Recoverer = (*ExchangeWalletSPV)(nil)
 var _ asset.PeerManager = (*ExchangeWalletSPV)(nil)
 var _ asset.TxFeeEstimator = (*intermediaryWallet)(nil)
-var _ asset.Bonder = (*baseWallet)(nil)
+var _ asset.Bonder = (*intermediaryWallet)(nil)
```

Comment:
This excludes the electrum wallet. Should I make a separate interface and not add to Bonder? The problem is `FindBond` is currently fetching blocks as a last resort, which I guess the Electrum wallet cannot currently do.

Reply:
You could make the electrum wallet do the expected path and not do the block fetching.

Reply:
Changed it so that the intermediary wallet will pull blocks but the base wallet will not.